lua/luarocks-tag-release.lua

```diff
@@ -55,30 +66,41 @@ local function luarocks_tag_release(args)
     return stdout, stderr
   end
 
-  ---@param rockspec_content string
+  local tmp_dir = execute('mktemp -d'):gsub('\n', '')
+
+  ---@param interpreter lua_interpreter
+  ---@return nil
+  local function luarocks_test(interpreter)
+    print('Initialising luarocks project...')
+    execute('rm -r lua_modules .luarocks luarocks lua', print)
```

Comment:
wouldn't it be preferable to do this in alternate folder rather than removing files (=scary) ?

Reply:
These files are created by `luarocks init` and added to a `.gitignore`.
They shouldn't be under version control. But I could try to init the project in a temporary directory.